searchlib/src/vespa/searchlib/expression/documentfieldnode.cpp

```diff
@@ -38,7 +38,7 @@ DocumentFieldNode::operator = (const DocumentFieldNode & rhs)
 {
     if (this != &rhs) {
         DocumentAccessorNode::operator=(rhs);
-        _fieldPath = rhs._fieldPath;
+        _fieldPath.clear();
```

Comment:
Is `_fieldPath` assigned elsewhere? Or is it not needed?

Reply:
Yes, in DocumentFieldNode::onDocType
...
docType.buildFieldPath(_fieldPath, _fieldName);
.....